internal/scanner/scanner.go

```diff
@@ -367,30 +369,37 @@ func (s *Scanner) produceTests(ctx context.Context, n int) <-chan *testWork {
 	go func() {
 		defer close(testChan)
 
-		var testHeaderValue string
+		var debugHeader string
```

Comment:
It is not a `debugHeader`, it is a `debugHeaderValue` if I understood it correctly. This name (`debugHeader`) confuses me because first I thought that it is the header (key) itself, but it can be key, value, key: value pair, anything. Please, add a suffix such as `Value`, `Key`, `Name`, anything to make it easier to understand

Reply:
You are right. Fixed - https://github.com/wallarm/gotestwaf/pull/160/commits/d16dbe242dcef003e9e4423b5f2a2fa50bc217d8